ENIGMAsystem/SHELL/Graphics_Systems/OpenGLES/Makefile

```diff
@@ -1,7 +1,7 @@
-SOURCES += $(wildcard Graphics_Systems/OpenGLES/*.cpp) $(wildcard Graphics_Systems/General/*.cpp)
-override CXXFLAGS += -IBridges/OpenGLES
+SOURCES += $(wildcard Graphics_Systems/OpenGLES/*.cpp) $(wildcard Graphics_Systems/OpenGLES/wrappers/*.cpp) $(wildcard Graphics_Systems/General/*.cpp)
 
-override LDLIBS += -lepoxy
+override CXXFLAGS += -IBridges/OpenGLES -IGraphics_Systems/OpenGLES/wrappers
+override CFLAGS += -I$(ENIGMA_ROOT)/android/external/glm

```

Comment:
Shouldn't this be moved into a `TARGET-PLATFORM` Android check?

Reply:
@RobertBColton I think I will look into this after GSoC